wherehows-web/app/components/search-bar-form.js

```diff
@@ -56,6 +56,18 @@ export default Component.extend({
       });
     },
 
+    /**
+     * Triggers a search action by the user pressing enter on a typeahead suggestion
+     * @param {string} suggestion - suggestion text passed in from aupac-typeahead
+     */
+    selectedSuggestion(suggestion) {
+      set(this, 'search', suggestion);
+      get(this, 'didSearch')({
```

Comment:
Rather than duplicate logic here, preferable to just invoke the `search` action directly from here
`this.actions.search.call(this)`

Also maybe rename this to `onSelectedSuggestion` since it's more descriptive of handling an event.

Reply:
Good point!